documentation/src/main/resources/pages/ditto/protocol/examples/things/generated/commands/modify/createthingresponse.md

```diff
@@ -22,7 +22,8 @@
           "y": 2.718,
           "z": 1,
           "unit": "g"
-        }
+        },
+        "desiredProperties": {}
```

Comment:
That looks wrong - in the related `creatething.md`, there was a `desiredProperty` "x" set to "4".
That must also be in the response of the created Thing.

Reply:
still missing as far as I see

Reply:
Yes you're right... Thought this was already fixed.
https://github.com/eclipse/ditto/pull/865/commits/1af9fc6397b3edb1b5c4de67e2a810a651e17291